### PR TITLE
Copy of UCBVI from rlberry_research to rlberry_scool and misc changes to doc

### PR DESCRIPTION
**Pull request by JulienT01**
add quick starts in user guide
add UCBVI in scool
Update quick start accordingly. Objective user guide rlberry-research-free.

## Checklist
- \[x] My code follows the style guideline
To check :
   black --check examples rlberry *py
   flake8 --select F401,F405,D410,D411,D412 --exclude=rlberry/check_packages.py --per-file-ignores="__init__.py:F401",
- \[ ] I have commented my code, particularly in hard-to-understand areas,
- \[x] I have made corresponding changes to the documentation,
- \[ ] I have added tests that prove my fix is effective or that my feature works,
- \[x] New and existing unit tests pass locally with my changes,
- \[x] If updated the changelog if necessary,
- \[x] I have set the label "ready for review" and the checks are all green.

### CODE REVIEW

**Comment by TimotheeMathieu** [comment]
Just a styling comment: maybe you could use toctrees instead of markdown links for user guide. Example:
```
```{toctree}
examples/content_child1.md
examples/content_child2.md
```
```

File: docs/basics/comparison.md [hunk 1]
```diff
@@ -116,12 +122,12 @@ print(compare_agents(agents))
...
 ```
 
-The results of `compare_agents(agents)` show the p-values and significance level if the method is `tukey_hsd` and in all the cases it shows the decision accept or reject of the test with Family-wise error controlled by $0.05$. In our case, we see that A2C seems significantly worst than both PPO and DQN but the difference between PPO and DQN is not statistically significant. Remark that no significance (which is to say, decision to accept $H_0$) does not necessarily mean that the algorithms perform the same, it can be that there is not enough data.
+The results of `compare_agents(agents)` show the p-values and significance level if the method is tukey_hsd and  it shows the decision accept or reject of the test with Family-wise error controlled by $0.05$. In our case, we see that DQN is worth than A2C and PPO but the difference between PPO and A2C is not statistically significant. Remark that no significance (which is to say, decision to accept $H_0$) does not necessarily mean that the algorithms perform the same, it can be that there is not enough data (and it is likely that it is the case here).
```
Comment (TimotheeMathieu) [inline docs/basics/comparison.md:131]: typo: DQN is worse than A2C and PPO

File: docs/user_guide.md [hunk 2]
```diff
@@ -8,28 +8,34 @@ Welcome to rlberry.
...
 ## Set up an experiment
-- [Environment](environment_page)
-- [Agent](agent_page)
-- [ExperimentManager](experimentManager_page)
-- [Logging](logging_page).
+```{include} templates/nice_toc.md
+```
+
+```{toctree}
+:maxdepth: 2
```
Comment (TimotheeMathieu) [inline docs/user_guide.md:33]: Do this for each of the sub menus

File: docs/basics/userguide/experimentManager.md [hunk 3]
```diff
@@ -6,13 +6,14 @@ It's the element that allows you to make your experiments on [Agent](agent_page)
 You can use it to train, optimize hyperparameters, evaluate, compare, and gather statistics about your agent on a specific environment. You can find the API doc [here](rlberry.manager.ExperimentManager).
 It's not the only solution, but it's the compact (and recommended) way of doing experiments with an agent.
 
-For these examples, you will use the "PPO" torch agent from "[rlberry-research](https://github.com/rlberry-py/rlberry-research)"
+For this examples, you will use the "PPO" torch agent from "[StableBaselines3](https://stable-baselines3.readthedocs.io/en/master/guide/algos.html)" and wrap it in rlberry Agent. To do that, you need to use [StableBaselinesAgent](rlberry.agents.stable_baselines.StableBaselinesAgent). More information [here](stable_baselines).
```
[the remark this answers is not in the document]
Comment (JulienT01) [reply docs/basics/userguide/experimentManager.md:9]: done

File: docs/basics/DeepRLTutorial/TutorialDeepRL.md [hunk 4]
```diff
@@ -313,57 +146,35 @@ Evaluating ...
 :align: center
 ```
 
-Let's try to change the neural networks' architectures and see if we can
-beat our previous result. This time we use a smaller learning rate and
-bigger batch size to have more stable training.
+Let's try to change the hyperparameters and see if we can
+beat our previous result. This time we use the recommanded hyperparameters from [rl-baselines3-zoo](https://github.com/DLR-RM/rl-baselines3-zoo/).
```
[the remark this answers is not in the document]
Comment (JulienT01) [reply docs/basics/DeepRLTutorial/TutorialDeepRL.md:150]: done

File: docs/basics/comparison.md (hunk 1 above)
Comment (JulienT01) [reply docs/basics/comparison.md:131]: done

File: docs/user_guide.md (hunk 2 above)
Comment (JulienT01) [reply docs/user_guide.md:33]: done

File: docs/basics/DeepRLTutorial/TutorialDeepRL.md [hunk 5]
```diff
@@ -451,6 +222,7 @@ INFO: Making new env: CartPole-v1
 ```{image} output_9_3.png
 :align: center
 ```
+Here, we can see that modifying the hyperparameters has accelerated learning...
```
[the remark this answers is not in the document]
Comment (JulienT01) [reply docs/basics/DeepRLTutorial/TutorialDeepRL.md]: done

**Comment by TimotheeMathieu** [comment]
LGTM for most of it. I did not review the quick start @AdrienneTuynman can you review this? 
The idea is to make comments either here or using the "Files changed" tab of the PR, based on what you see in the preview of the documentation that can be found here: https://rlberry-py.github.io/rlberry/preview_pr
